about.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="styles.css">
+  <title>About</title>
+</head>
+<body>
+  <section class="first">
+    <header>
+      <div class="flists">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-facebook-f"></i>
+        <ul class="flist">
+          <li><a href="">English</a></li>
+          <li><a href="">My page</a></li>
+          <li><a href="">logout</a></li>
+        </ul>
+      </div>
+      <div class="bar" id="bar">
+        <i class="fa-solid fa-bars fa-2x"></i>
+      </div>
+      <div class="hide" id="close">
+        <i class="fa-solid fa-xmark "></i>
+      </div>
+      <nav class="lnav">
+        <div class="logo">
+          <h1><a href="index.html">AYC</a></h1>
+        </div>
+        <div >
+          <ul class="barlist">
+            <li><a href="">About</a></li>
+            <li><a href="">Program</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Sponsor</a></li>
+            <li><a href="">News</a></li>
+            <li><button>AYC company</button></li>
+          </ul>
+      </div>
+      </nav>  
+    
+    </header>
+    <div class="hi" id="hi">
+      <h2>African Youth Confrance 2022</h2>
+      <p>As a National Resource Center for African Studies, the Harvard University Center for African Studies compiles Africa-related seminars, workshops, events, and conferences taking place in Cambridge, Boston, and Johannesburg, South Africa. We also include virtual Africa-related events and activities that are free and open to the public. The Center for African Studies will consider requests to host, coordinate, sponsor, or co-sponsor Africa-related events only if the event is supported by one of our faculty affiliates.</p>
+      <h3 class="date">July 27 ~ 28</h3>
+      <h3 class="place">@ University of Notre Dame.</h3>
+    </div>
+    </section>
```

Comment:
- [ ] Please include all this code inside one `<section>` tag and make the page `<header>` outside this section for semantic reasons.